### ISSUE TITLE
Any plans to add Scala Kafka Admin Client?

### ISSUE THREAD

**Issue by EddyP23** [cakesolutions/scala-kafka-client#123]
From what I see looking at the repo, currently you have wrappers for Consumer and Producer, but I think it would be nice to have a wrapper for `KafkaAdminClient` as well

**Comment by simonsouter** [comment]
Yes I have been thinking about that.  Its not too painful to use the java classes directly, but I will look if it's worth wrapping them with some Scala helpers.

PRs welcome of course :)
